### ISSUE TITLE
Sandman 0.5.2a Box Group(s) issue

### ISSUE THREAD

**Issue by Coverlin** [sandboxie-plus/Sandboxie#262]
1
Having a Group and a box with identical names will cause sandman to list the box repeatedly.
example : BoxDisplayOrder=DefaultBox,Test(Test,Test2)

2
Disable forced programs input is always default to '10' seconds ignoring existing 'ForceDisableSeconds'
The input takes up to 3 characters from keyboard (max value: 999) goes above if the increment arrow is used , Sbiectrl takes about 9 characters
Workaround : Tray context menu(toggle) or using value 0 in sandman's inputbox won't change the current value

3a
Sandman box options -> file options -> file migration
Doesn't reflect the new CopyLimitKb=-1 setting (box) although the -1 value can be set from the UI (shows the default value of 80mb and unchecked)
In short: can set but doesn't show -1

3b
Sbiectrl file migration does show if the box uses the CopyLimitKb=-1 but the old UI does not accept typing non number characters (copy and paste of the -1 value works but also turns it into CopyLimitKb=999999999 after apply)
In short: cannot set but shows -1

4  Suggestion
Sandman new sorting to use the same layout as Sbiectrl (BoxDisplayOrder)
Currently the sorting is column ascending/descending so maybe have a 3rd option when clicking a column(Name) to switch to the custom boxdisplayorder layout

**Comment by DavidXanatos** [comment]
1. will try out 

2. that's the problem you want more than 999? wil change it to nine 9's. I can NOT can not that it would ignore the value, to me when i choose 3 sec its 3 sec then it set 999 it just stays off longer than i'm willing to wait

3. a) thats how it supposed to work when its -1 in the ini the checkbox gets unchecked and the edit field shows the default value
3. b) will check that out

4. switching to custom order would imply that i also add a mechanism to change that order may by later

**Comment by Coverlin** [comment]
1
Alternatively creating the Test group and then trying to create a Test box will replace the group with the box
Recreating the Test group will trigger the Test box to be listed repeatedly
restarting Sandman in this state will only have the Test box (and the defaultbox)
-----
There is a new bug with creating a group with sandman(0.5.2a) , even after creating it no sandbox can be assigned/moved to it
Does not create the BoxDisplayOrder entry if a new config is used , affects new users.

2
yes , fixed in 0.5.2 / 5.45.1
Regarding the 10 s i meant that the imput box would start always with 10 s regardles of current value of ForceDisableSeconds, it's no longer a issue with the character limit lifted.

3a
I don't know why i thought that the value -1 should be shown when just uncheking the box does the same thing as setting the value -1 (tbh i typed the value in ...lol) ... this point was a invalid issue mb

3b
Sbiectrl allowing only numbers to be typed restricion was lifted with 0.5.2 / 5.45.1 release

Thank you !

**Comment by DavidXanatos** [comment]
mist that one for the 0.5.3a pre release but will be fixed in the next 0.5.3b release
